x-pack/plugins/maps/public/components/map/feature_tooltip.js

```diff
@@ -14,11 +14,13 @@ export class FeatureTooltip extends React.Component {
 
   _renderProperties() {
     return Object.keys(this.props.properties).map(propertyName => {
+      // eslint-disable-next-line react/no-danger
+      const htmlValue = <span dangerouslySetInnerHTML={{ __html: this.props.properties[propertyName] }} />;
```

Comment:
Can you make this a multiline statement so its easier to read. Also, all of the other uses of dangerouslySetInnerHTML provide a detailed comment about how the risk is mitigated. For example https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/metric_vis/public/components/metric_vis_value.js#L59. I think we need to do the same here

```
const htmlValue = (
  <span 
    /*
     * Justification for dangerouslySetInnerHTML:
     * Propery value contains value generated by Field formatter
     * Since these formatters produce raw HTML, this component needs to be able to render them as-is, relying
     * on the field formatter to only produce safe HTML.
     */
    dangerouslySetInnerHTML={{ __html: this.props.properties[propertyName] }} 
  />
);
```